README.md

```diff
@@ -1,4 +1,5 @@
 [![Latest Release](https://img.shields.io/github/v/release/sda-se/sda-dropwizard-commons?label=latest)](https://github.com/SDA-SE/sda-dropwizard-commons/releases/latest)
+[![javadoc](https://javadoc.io/badge2/org.sdase.commons/sda-commons-bom/javadoc.svg)](https://javadoc.io/doc/org.sdase.commons/sda-commons-bom)
```

Comment:
I'm not sure if the bom is the right one to choose here. At least it doesn't provide a javadoc so the linked resource will not work :-) Maybe sda-commons-server-starter?

Reply:
How about keeping the badge image as-is and changing the link to: https://javadoc.io/doc/org.sdase.commons
However that content looks wrong?

Reply:
Looks like the contents of that page are filled ones they are generated. Maybe we should provide a link in every sub-readme to initialize them ;)